examples/nidaqmx_analog_input/NIDAQmxAnalogInput.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "ni.examples.NIDAQmxAnalogInput_Python",
       "descriptionUrl": "",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "NI-DAQmx Analog Input Example",
```

Comment:
Provide a short description of the example, not just a reformatting of the name.  Using the top line comment of each example's measurement.py would be sufficient.